.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        entry:
+          - { ruby: "2.7" }
+          - { ruby: "2.7", concurrency: async-websocket }
+          - { ruby: "3.0" }
+          - { ruby: "3.1" }
+          - { ruby: ruby-head, ignore: true }
+          - { ruby: jruby-head, ignore: true }
```

Comment:
I see how `ignore` allows the build to pass later in the `steps` but I'm wondering is it simply best practice to test against the heads?

Reply:
It's there to catch bugs early, that's all.